vitrina/projects/templates/vitrina/projects/detail.html

```diff
@@ -11,6 +11,10 @@
 {% endblock %}
 
 {% block content %}
+    {% if can_manage_history %}
```

Comment:
I suggest to move this check into `component.tabs.html`, because, there will be more tabs in the future.